pandas/core/indexes/base.py

```diff
@@ -2529,15 +2529,20 @@ def set_value(self, arr, key, value):
     def _get_level_values(self, level):
         """
         Return an Index of values for requested level, equal to the length
-        of the index
+        of the index.
```

Comment:
**Not Blocking** : I would really like if we could condense this into one line (doc-string convention), but it's hard to see ATM what to remove from this...

Reply:
I haven't got a better proposa either.